netcdf.yaml

```diff
@@ -0,0 +1,80 @@
+package:
+  name: netcdf
+  version: 4.9.2
+  epoch: 0
+  description: Libraries for the Unidata network Common Data Format
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - bzip2-dev
+      - cmake
+      - curl-dev
+      - hdf5-dev
+      - libaec-dev
+      - libxml2-dev
+      - m4
+      - samurai
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Unidata/netcdf-c
+      tag: v${{package.version}}
+      expected-commit: 9328ba17cb53f13a63707547c94f4715243dafdf
+
+  - uses: patch
+    with:
+      patches: gcc14.patch
```

Comment:
I don't see this patch associated with this PR. 💭 Is this required for building this package?